libsmtutil/SMTPortfolio.cpp

```diff
@@ -26,10 +26,10 @@ using namespace solidity::frontend;
 using namespace solidity::smtutil;
 
 SMTPortfolio::SMTPortfolio(
-	std::vector<std::unique_ptr<SolverInterface>> _solvers,
+	std::vector<std::unique_ptr<BMCSolverInterface>> _solvers,
```

Comment:
No, because before this PR `CHCSolverInterface` was not a subclass of `SolverInterface`.
`SolverInterface` was in fact an interface for a solver used by the BMC engine.